appveyor.yml

```diff
@@ -13,10 +12,15 @@ install:
 #- choco install tfs2010objectmodel -y
 #- choco install tfs2012objectmodel -y
 #- choco install tfs2013objectmodel -y
-#build:
-#  project: GitTfs.sln
-#  verbosity: minimal
 build_script:
-  msbuild CI.proj /t:Build /nologo /verbosity:minimal /p:BuildInParallel=true /m:4
+  - msbuild GitTfs.sln /t:GitTfs_Vs2015 /t:GitTfsTest /verbosity:minimal /p:Configuration=Debug /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /nologo /p:BuildInParallel=true /m:4
+after_build:
+  - ps: |
+      $libgit2Binaries=Get-ChildItem ".\packages\" -Filter "NativeBinaries" -Recurse | Select-Object -First 1 | % { $_.FullName }
+      Copy-Item $libgit2Binaries ".\GitTfs.Vs2015\bin\Debug\NativeBinaries" -Recurse
+
 test:
-  assemblies: '**\GitTfsTest.dll'
\ No newline at end of file
+  assemblies: '**\GitTfsTest.dll'
+after_test:
+  - ps: |
+      & ".\FunctionalTesting\smoke_tests.ps1" "$Env:APPVEYOR_BUILD_FOLDER\GitTfs.Vs2015\bin\Debug"
```

Comment:
Why not run from `$Env::APPVEYOR_BUILD_FOLDER\GitTfs\bin\Debug`? Won't that also exist, and already have the libgit2 native binaries?